flow/templates/expanse.sh

```diff
@@ -0,0 +1,54 @@
+{# Templated in accordance with: https://www.sdsc.edu/support/user_guides/expanse.html #}
+{% extends "slurm.sh" %}
+{% block tasks %}
+{% set threshold = 0 if force else 0.9 %}
```

Comment:
Please indent this script for readability. See #495 or the [Bridges2 template](https://github.com/glotzerlab/signac-flow/blob/master/flow/templates/bridges2.sh) for examples.